SJMaster/urls.py

```diff
@@ -16,35 +16,29 @@
 from django.contrib import admin
 from django.urls import path, include
 from jobboard.views import board
+from recruiter.views import update_recruiter_account_settings_view
 from student.views import update_student_account_settings_view, account_update_success
-from recruiter.views import (UpdateRecruiterSettings,
-                             CreateNewJobForm,
-                             job_created_successfully,
-                             recruiterRegister,
-                             recruiter_created_successfully,
-                             recruiter_view_my_jobs_and_applications,)
+from recruiter.views import (
+    CreateNewJobForm,
+    job_created_successfully,
+    recruiterRegister,
+    recruiter_created_successfully,
+    recruiter_view_my_jobs_and_applications, )
```

Comment:
what's this space for?

Reply:
looks like a typo. fixed it in PR #69 so when it will be merged it'll be fixed